tcp_check/datadog_checks/tcp_check/tcp_check.py

```diff
@@ -51,6 +51,10 @@ def __init__(self, name, init_config, instances):
         if not isinstance(self.host, str):  # Would be raised if url is not a string
             raise ConfigurationError(self.CONFIGURATION_ERROR_MSG.format(self.host, 'url', 'string'))
 
+        scope_id = instance.get('scope_id', 0) #Get the interface ID that should handle the traffic (necessary for IPv6 only)
```

Comment:
```suggestion
        self.scope_id = instance.get('scope_id', 0) #Get the interface ID that should handle the traffic (necessary for IPv6 only)
```
Currently failing CI due to this